superset-frontend/src/SqlLab/components/TabbedSqlEditors/index.jsx

```diff
@@ -401,6 +409,19 @@ class TabbedSqlEditors extends React.PureComponent {
       );
     });
 
+    const emptyTab = (
+      <EditableTabs.TabPane
+        key={Math.random()}
+        data-key={Math.random()}
+        tab={<TabTitle>Add a New Tab</TabTitle>}
+        closable={false}
+      >
+        <StyledImage>
+          <img src={emptySqlChart} alt="No SQL tabs" />
```

Comment:
can you write a small test making sure this renders when `this.props.queryEditors?.length === 0`

Reply:
will do rn